dataprofiler/profilers/profiler_options.py

```diff
@@ -396,7 +450,9 @@ def __init__(self) -> None:
         self.median_abs_deviation: BooleanOption = BooleanOption(is_enabled=True)
         self.num_zeros: BooleanOption = BooleanOption(is_enabled=True)
         self.num_negatives: BooleanOption = BooleanOption(is_enabled=True)
-        self.histogram_and_quantiles: HistogramOption = HistogramOption()
+        self.histogram_and_quantiles: HistogramAndQuantilesOption = (
```

Comment:
Below in validate, do we need to fix the validate? Right now, it is validating it is a BooleanOption, while true, that would cause failures elsewhere if wasn't of this class bc of variable references.